mesonpy/__init__.py

```diff
@@ -309,10 +309,12 @@ def __init__(
         metadata: Metadata,
         manifest: Dict[str, List[Tuple[pathlib.Path, str]]],
         limited_api: bool,
+        windows_shared_libs: bool,
     ) -> None:
         self._metadata = metadata
         self._manifest = manifest
         self._limited_api = limited_api
+        self._windows_shared_libs = windows_shared_libs
```

Comment:
Allow `allow_` into this name may be useful? If you considered and rejected it based on the name being too long, then the current name is fine with me.